apps/app/src/app/core/services/settings.service.spec.ts

```diff
@@ -14,6 +15,25 @@ describe('SettingsService', () => {
     localStorage.removeItem('currentApplications');
```

Comment:
Je ne pense pas que ce soit une bonne idée d'utiliser le vrai localStorage car tu veux tester ici tes fonctionnalités du service sans dépendre de la disponibilité du localStorage. Pour cela tu devrais le mocker pour pouvoir l'utiliser. J'ai trouvé un exemple plutôt bien explique ici : https://armno.medium.com/til-mocking-localstorage-and-sessionstorage-in-angular-unit-tests-a765abdc9d87

Reply:
Oh, c'est super intéressant ça !